doc/getting_started/index.rst

```diff
@@ -61,18 +51,13 @@ tools. First, make sure your development system OS is updated:
 Install dependencies
 ********************
 
-Next, use a package manager to install required support tools. Python 3
-and its package manager, pip, are used extensively by Zephyr for
-installing and running scripts used to compile, build, and run Zephyr
-applications.
-
-We'll also install Zephyr's multi-purpose west tool.
+Next, you'll install some host dependencies using your package manager.
```

Comment:
Maybe emphasize the importance ?
```suggestion
Next, you'll install required host dependencies using your package manager.
```

Reply:
Well, they are not all required for all use cases, as discussed earlier in the review. Strictly speaking several of these are optional dependencies only needed for some targets or some use cases.

Since I was requested to leave all the optional dependencies I discovered in thisdocument, I have done so, but they're not required ;).

Reply:
For reference and the record:

- ccache: this is an optimization
- wget: only needed to pull down the Zephyr SDK, which is optional
- make: optional
- gcc, gcc-multilib, g++-multilib: only needed for some host targets, per review
- dfu-util: only needed for some boards as a runner dependency
- libsdl2-dev: I honestly don't know
- python3-tk: only needed for the guiconfig target, as far as I can tell
- file: only needed by the SDK installer
- xz-utils: needed to unpack the SDK, not required on all toolchains
- device-tree-compiler: optional, just used for additional warnings


You can see that not much is actually required.

Reply:
Thanks for the list.
Could we add this information somewhere and link to it ?

Personally I like to be able to know why I need to install a certain tool, if it is not strictly required. 
(Note: this can be an independent improvement to the cleanup)

Reply:
I'd rather not maintain such a thing and it's likely to go stale. People tend to build additional users on existing dependencies.

Reply:
point taken.